core/arch/arm/plat-stm32mp1/nsec-service/stm32mp1_svc_setup.c

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/*
+ * Copyright (c) 2017-2018, STMicroelectronics
+ */
+
+#include <arm.h>
+#include <drivers/stm32_bsec.h>
+#include <sm/optee_smc.h>
+#include <sm/sm.h>
+#include <string.h>
+#include <stm32_util.h>
+
+#include "bsec_svc.h"
+#include "stm32mp1_smc.h"
+
+/* STM32 SiP Service UUID */
+static const TEE_UUID stm32mp1_sip_svc_uid = {
+	0x50aa78a7, 0x9bf4, 0x4a14,
```

Comment:
Why isn't this defined in `stm32mp1_smc.h` in a manner similar to what we do in `optee_msg.h`?

Reply:
Right, will do. thanks.